### PR TITLE
Bugfix/adminblip

### PR DESCRIPTION
**Pull request by poulpito**
linked to issue #114 

Change the method to remove all blip (native does not work - using QBCore removeblip instead)
and add a loop to check if a player disconnect during all the blip updates.

### CODE REVIEW

**Review by NariieL — APPROVED** [review]
Approved

File: resources/[soz]/soz-core/src/client/admin/admin.menu.interactive.provider.ts [hunk 1]
```diff
@@ -101,22 +101,28 @@ export class AdminMenuInteractiveProvider {
...
-            }
-            clearInterval(this.intervalHandlers.displayPlayersOnMap);
-            return;
+        this.playerBlips.forEach((BlipValue, BlipKey) => {
+            this.QBCore.removeBlip('admin:player-blip:' + BlipKey);
+            this.playerBlips.delete(BlipKey);
+        });
+
+        clearInterval(this.intervalHandlers.displayPlayersOnMap);
+        return;
```
Comment (joelwurtz) [inline resources/[soz]/soz-core/src/client/admin/admin.menu.interactive.provider.ts]: problème d'indentation non ?

**Comment by poulpito** [comment]
Rebase & fix ident as suggested.

**Comment by joelwurtz** [comment]
It has been merged into our repository see https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/commit/13f4ab2369d006d1c2c9a086a5769a5f14afb4a6 

Thanks a lot ❤️

**Pull request closed without merging**